src/main/reactjs/src/components/publicTranslator/PublicTranslatorFilters.tsx

```diff
@@ -60,6 +62,19 @@ export const PublicTranslatorFilters = ({
     filters: reduxFilters,
   } = useAppSelector(publicTranslatorsSelector);
 
+  const hasError = (fieldName: SearchFilter) => {
+    return reduxFilters?.errors?.includes(fieldName);
+  };
+
+  // Adhere to Koodisto lang codes (uppercase)
+  const fromLanguages = langs.from.map((langCode: string) =>
+    langCode.toUpperCase()
+  );
+  const toLanguages = langs.to.map((langCode: string) =>
+    langCode.toUpperCase()
+  );
+  const languages = { from: fromLanguages, to: toLanguages };
```

Comment:
Tällä hetkellä lang koodit taisi olla uppercasena redux tilassa, tarviiko näitä vielä?

Reply:
Lang koodit palautuu vielä reduxista lowercasena; Rivi 60: langs